samples/cpp/benchmark_app/main.cpp

```diff
@@ -249,34 +249,33 @@ int main(int argc, char* argv[]) {
             auto& device_config = config.at(device);
 
             // high-level performance modes
-            if (!ov_perf_hint.empty()) {
-                device_config[CONFIG_KEY(PERFORMANCE_HINT)] = ov_perf_hint;
+            if (ov_perf_hint != ov::hint::PerformanceMode::UNDEFINED) {
+                device_config.emplace(ov::hint::performance_mode(ov_perf_hint));
                 if (FLAGS_nireq != 0)
-                    device_config[CONFIG_KEY(PERFORMANCE_HINT_NUM_REQUESTS)] = std::to_string(FLAGS_nireq);
+                    device_config.emplace(ov::hint::num_requests(FLAGS_nireq));
             }
 
             // Set performance counter
             if (isFlagSetInCommandLine("pc")) {
                 // set to user defined value
-                device_config[CONFIG_KEY(PERF_COUNT)] = FLAGS_pc ? CONFIG_VALUE(YES) : CONFIG_VALUE(NO);
-            } else if (device_config.count(CONFIG_KEY(PERF_COUNT)) &&
-                       (device_config.at(CONFIG_KEY(PERF_COUNT)).as<std::string>() == "YES")) {
+                device_config.emplace(ov::enable_profiling(FLAGS_pc));
+            } else if (device_config.count(ov::enable_profiling.name()) &&
+                       (device_config.at(ov::enable_profiling.name()).as<bool>())) {
```

Comment:
Can we use without name() here?

Reply:
No. `device_config` is just `AnyMap` which  is just alias for `std::map<std::string, ov::Any>`